api/src/main/java/org/openmrs/module/metrics/MetricEvent.java

```diff
@@ -1,38 +1,62 @@
 package org.openmrs.module.metrics;
 
-import java.io.Serializable;
-import java.net.URI;
-import java.net.URISyntaxException;
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
 import java.time.LocalDateTime;
+import java.util.Date;
 
-import org.openmrs.BaseOpenmrsObject;
+import org.openmrs.BaseOpenmrsData;
 
-public class MetricEvent extends BaseOpenmrsObject implements Serializable {
-	
-	private int id;
-	
+@Entity(name = "metric.Event")
+@Table(name = "metric_event_records")
+@Inheritance(strategy = InheritanceType.JOINED)
+public class MetricEvent extends BaseOpenmrsData {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@Column(name = "metric_event_id")
+	private Integer id;
+	
+	@Basic
+	@Column(name = "title")
 	private String title;
 	
-	private LocalDateTime timeStamp;
+	@Basic
+	@Column(name = "timeStamp")
```

Comment:
Shouldn't this be `"time_stamp"` because of how we changed things in the Liquibase file?

Reply:
yes, it should be time_stamp